hasura.planx.uk/metadata/cron_triggers.yaml

```diff
@@ -1,3 +1,12 @@
+- name: analyze_sessions
+  webhook: '{{HASURA_PLANX_API_URL}}/webhooks/hasura/analyze-sessions'
+  schedule: '* * * * *'
```

Comment:
This is running every minute for easy testing on the pizza - what's our preferred schedule for staging/production? Nightly? More than once a day?

Reply:
I think nightly to start with seems reasonable, as far as I'm aware there aren't any users / use cases where the lag between submission and processing would be an issue.

Reply:
Updated to run once daily at 3am!